### ISSUE TITLE
why Invalid schema error with ingressclassparams_v1beta1.json ? 

### ISSUE THREAD

**Issue by HanHoRang31** [datreeio/CRDs-catalog#88]
I am trying to validate the schema through the schema of CRDs-catalog/elbv2.k8s.aws with kubectl-datree. 
When entering the following command, an invalid schema error occurs. 
Is there something I'm missing?

### input command
`kubectl datree test --schema-location ingressclassparams_v1beta1.json -- --all
`
### output
<img width="832" alt="image" src="https://user-images.githubusercontent.com/117359361/209917652-3d7df954-f874-49f9-a2cd-dafcff5b8577.png">

**Comment by hadar-co** [comment]
Hi @HanHoRang31 , the `--schema-location` flag expects a path or url, you provided a filename (which I assume it will look for in the directory you're running the test in).
In order to use the schemas in the CRD catalog, you don't need to specify a `--schema-location`, the CLI will automatically look in the catalog for the needed schemas.

Can you try to remove the `--schema-location` from your command and run again?

**Comment by eyarz** [comment]
@hadar-co I don't think this is the case here because the kind `ingressclassparams` is recognized.

@HanHoRang31 I suspect you have an indentation error around the `spec` property.
can you share your manifest so we can tell for sure?

**Comment by HanHoRang31** [comment]
Thanks for the reply
@hadar-co  
I tried, but the result is the same

@eyarz 
It's hard to share. I'm testing for datree the yaml in prod EKS, so there are a lot of shared files and personal information.
 

--
Looking at the error schema, it seems to be caused by two things.
<img width="969" alt="image" src="https://user-images.githubusercontent.com/117359361/210684499-d9e22064-7d16-4064-8898-52259dd8cfb0.png">

1. Problem recognizing kubectl-customresourcedefinition.apiextensions.k8s.io-targetgroupbindings.elbv2.k8s.aws.yaml
- I entered a command through a file provided by CRD(ingressclassparams_v1beta1.json). Should I enter another command?

2. datree doesn't seem to recognize the ipAddreeType of kubectl-targetgroupbinding.elbv2.k8s
